chardet/cli/chardetect.py

```diff
@@ -19,12 +19,11 @@
 import sys
 from io import open
 
-from chardet import __version__
-from chardet.compat import PY2
+from chardet.version import __version__
 from chardet.universaldetector import UniversalDetector
 
 
-
+PY_VER = 2 if sys.version_info < (3, 0) else 3
```

Comment:
it's common practice in Python to include checks like this in a `compat.py` module, which is what we were doing before. Please change this back to just using `PY2` instead of `PY_VER == 2`.